database/db.js

```diff
@@ -110,7 +110,7 @@ Property.insertMany(dummyData.propertyData, (err, data) => {
 });
 
 // Insert dummydata into comparableHomes model
-ComparableHomes.insertMany(dummyData.comparableHomeData, (err, data) => {
```

Comment:
Earlier, when I said that you should have seeding in one file, I meant that you should have a separate script named `seeder.js` or similarly, and have `npm run seed-data` run that instead. The way it's done here, every time the server is run, it re-seeds the collection with fake data.

Reply:
@leightonchen thanks for the tip. I made this change as well!